core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -187,6 +237,11 @@ public String getPrincipal() {
     return getCredentials().getPrincipal();
   }
 
+  public UncaughtExceptionHandler getUncaughtExceptionHandler() {
+    ensureOpen();
+    return ueh;
+  }
+
```

Comment:
Is this getter method ever used outside the constructor? If not, it can be removed, along with the field it gets.

Reply:
Method removed in a6236ae